packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.css

```diff
@@ -0,0 +1,10 @@
+.combobox-wrapper {
```

Comment:
To follow the same convention as SLDS classes (BEM style), I would suggest to use `sort` as the block name, and `container`/`header` as the elements. In this case the CSS classes would change like this:

`combobox-wrapper` --> `sort__container`
`combobox__sort-label` --> `sort__header`